app/Http/Controllers/ProfileController.php

```diff
@@ -34,11 +35,13 @@ public function store(Request $request){
 
     $user = User::create([
         'user_id'        => auth()->user()->id,
+
         'avatar'          =>  'data:image/' . $request->avatar->extension() . ';base64,' . base64_encode(file_get_contents($request->image)),
```

Comment:
please make this a file upload so and the path is the only inserted in the database